Cargo.toml

```diff
@@ -21,17 +21,16 @@ vm-memory = "0.7"
 vmm-sys-util = "0.9.0"
 tokio = { version = "1", features = ["macros"] }
 tokio-test = "0.4.2"
+tokio-uring = "0.3.0"
```

Comment:
This is unavailable for macos， so need to conditionally compile it for linux only.

Reply:
fixed